### DISCUSSION TITLE
Evaluating GovFlow as a back-end for managing grievances on a World Bank project

### DISCUSSION THREAD

**Discussion by robworthington** [govflow/govflow#76] [General]
Hello,

I am exploring use of Open311 as a protocol for managing grievances reported on World Bank projects.  There is a post on the Open 311 website introducing GovFlow as one open source option for receiving and managing issues.

I'd welcome any information to understand if you encourage third party usage and if so what documentation or support might be available.

Many thanks

Rob

**Comment by pwalsh** [comment]
Hey @robworthington govflow is fully usable as a server for 311. apart from the docs in the code base, there are no other docs, but feel free to email me any questions

**Comment by WillemvdW** [comment]
Hi, I work with Rob. Getting errors with running the installation. Please advise the prerequisites in the environment to do a fresh installation, such as node versions, database requirements etc.

**Reply by pwalsh** [reply]
@WillemvdW hi there 👋 

Node version is declared in `[.nvmrc](https://github.com/govflow/govflow/blob/main/.nvmrc)`

Any Postgres version should be fine. Apart from what you can find in https://github.com/govflow/govflow/tree/main/docs and the main README, there are no other docs, but feel free to reach out to me if you have specific questions
